File: backend/backend/Models/Comment.cs
namespace backend.Models
{
    public class Comment
    {
        public Guid Id { get; set; }
        public Post Post { get; set; }
        public string Content { get; set; }
        public DateTime Date { get; set; } = DateTime.UtcNow;
        public AppUser Author { get; set; } 
    }
}

File: client/src/app/models/Comment.ts
import { Author } from "./Post";

export interface Comment{
    id:string,
    postId:string,
    content:string,
    date:string,
    author:Author,
}

export interface CreateComment{
    postId:string,
    content:string,
}

export interface UpdateComment{
    content:string,
}

export interface DeleteComment{
    postId:string,
    commentId:string,
}